faust/assignor/partition_assignor.py

```diff
@@ -327,7 +327,10 @@ def _global_table_standby_assignments(
                         assignment.actives.get(changelog_topic_name, [])
                     )
                     # Only add those partitions as standby which aren't active
-                    standby_partitions = all_partitions - active_partitions
+                    if not table.is_global_global:
+                        standby_partitions = all_partitions - active_partitions
+                    else:
+                        standby_partitions = all_partitions
```

Comment:
How do you make sure that you don't have any `active_partitions` if you use a global global table?
Or is it allowed that standby and active_partitions are intersecting?

If not you would need to reset `all_partitions` to an empty set in the else branch, because it is used for the active partition assignment below

Reply:
I think it's ok to have a partition as active and standby. But this is probably to be tested in a real environment. The only "funny" thing is then that you write to an active and it's saved in the rocksdb and then it's send to the changelog as well and in the background because it's a standby it's received from the changelog again and written a second time.